README.adoc

```diff
@@ -198,16 +208,142 @@ curl http://localhost:9080/hello
 ```
 endif::[]
 
-=== Tearing down the Docker container
+After you are finished checking out the application, stop your container by running the following command:
+
+[role='command']
+```
+docker stop springBootContainer
+```
+
+ifndef::cloud-hosted[]
+== Faster starting up the container with InstantOn - Optional
+
+Open Liberty InstantOn provides fast startup times for MicroProfile and Jakarta EE applications. In this section, you’ll learn how to build an InstantOn image that provides fast startup times. Your application can start in milliseconds, without compromising on throughput, memory, development-production parity, or Java language features.
 
-To stop and remove your container, run the following commands:
+If you are working on a Linux system with kernel version 5.9 or greater, you can continue with this section; otherwise, go to the <<running-the-application-on-open-liberty>> section. You can run the following command to check out the kernel version of your system:
+
+[role='command']
+```
+uname -r
+```
+
+The https://openliberty.io/docs/latest/reference/feature/crac.html[Coordinated Restore at Checkpoint^] feature enables Liberty to use the Liberty’s implementation of https://javadoc.io/doc/org.crac/crac/1.4.0/index.html[org.crac^] APIs and more logs from Spring when the application is restoring.
+
+[role="code_command hotspot file=0", subs="quotes"]
+----
+#Replace the `Dockerfile` in the `start` directory.#
```

Comment:
This is confusing. Took me a minute to realize this means replace the `Dockerfile` content with the new content of the displayed `Dockerfile` in the tabs on the right.

Would it be more clear to have the title be:

### Enable the CRaC feature
Enable the `crac-1.4` feature with the adding the following `COPY` command in the `Dockerfile`:

```
COPY --chown=1001:0 src/main/liberty/instantOn/crac.xml \
                    /config/configDropins/defaults/crac.xml
```

Reply:
In the guides practice, we use hotspot to highlight the command/code and its location in the file, instead of just providing it in the content. I updated that paragraph. Hope it can help.